src/item.rs

```diff
@@ -1,80 +1,530 @@
+    ///     .unwrap();
+    ///
+    /// let category_2 = CategoryBuilder::new()
+    ///     .domain(Some("http://jupiterbroadcasting.com".to_owned()))
+    ///     .name("Podcast")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// let categories_vec = vec![category_1, category_2];
+    ///
+    /// let item = ItemBuilder::new()
+    ///     .title(Some("Making Music with Linux | LAS 408".to_owned()))
+    ///     .categories(categories_vec.clone())
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert_eq!(categories_vec.clone().len(), item.categories().len());
+    /// ```
+    pub fn categories(&self) -> Vec<Category> {
+        self.categories
+            .clone()
+    }
+
+    /// Get the optional comments that exists under `Item`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rss::{ItemBuilder, Item};
+    ///
+    /// let comments_string = "http://example.com/comments".to_owned();
+    ///
+    /// let item = ItemBuilder::new()
+    ///     .title(Some("Making Music with Linux | LAS 408".to_owned()))
+    ///     .comments(Some(comments_string.clone()))
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// let comments_option =  item.comments();
+    /// assert!(comments_option.is_some());
+    ///
+    /// assert_eq!(comments_string.clone(), comments_option.unwrap());
+    /// ```
+    ///
+    /// ```
+    /// use rss::{ItemBuilder, Item};
+    ///
+    /// let item = ItemBuilder::new()
+    ///     .title(Some("Making Music with Linux | LAS 408".to_owned()))
+    ///     .comments(None)
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(item.comments().is_none());
+    /// ```
+    pub fn comments(&self) -> Option<String> {
+        self.comments
+            .clone()
+    }
+
+    /// Get the optional enclosure that exists under `Item`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rss::{EnclosureBuilder, ItemBuilder, Item};
+    ///
+    /// let url = "http://www.podtrac.com/pts/redirect.ogg/".to_owned()
+    /// + "traffic.libsyn.com/jnite/linuxactionshowep408.ogg";
+    ///
+    /// let enclosure = EnclosureBuilder::new()
+    ///     .url(url.as_ref())
+    ///     .length(70772893)
+    ///     .mime_type("audio/ogg")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// let item = ItemBuilder::new()
+    ///     .title(Some("Making Music with Linux | LAS 408".to_owned()))
+    ///     .enclosure(Some(enclosure))
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(item.enclosure().is_some());
+    /// ```
+    ///
+    /// ```
+    /// use rss::{ItemBuilder, Item};
+    ///
+    /// let item = ItemBuilder::new()
+    ///     .title(Some("Making Music with Linux | LAS 408".to_owned()))
+    ///     .enclosure(None)
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(item.enclosure().is_none());
+    /// ```
+    pub fn enclosure(&self) -> Option<Enclosure> {
```

Comment:
The clones can be removed for the getters that return other structs as well. This could return `Option<&Enclosure>` by calling `as_ref()` on `self.enclosure`.